translate/lang/data.py

```diff
@@ -102,6 +102,7 @@
     'jbo': ('Lojban', 1, '0'),
     'jv': ('Javanese', 2, '(n != 1)'),
     'ka': ('Georgian', 1, '0'),
+    'kab': ('Kabyle', 1, '(n > 1)'),
```

Comment:
Yeah that's right. Sorry. due to copy past.
The right form is as you wrote: 'kab': ('Kabyle', 2, '(n != 1)'),
Shoud I resend an other pull?